### ISSUE TITLE
Failed to run query

### ISSUE THREAD

**Issue by SteveRoland** [laughingman7743/PyAthenaJDBC#2]
## Description

I just ran the sample code in the 'readme.md'.

I get the folowing exception:

```
Traceback (most recent call last):
  File "handler.py", line 17, in <module>
    """)
  File "/Users/skyfall/Documents/workspace/test/python/pyathena/vendor/pyathenajdbc/cursor.py", line 113, in execute
    reraise_dbapi_error()
  File "/Users/skyfall/Documents/workspace/test/python/pyathena/vendor/pyathenajdbc/util.py", line 27, in reraise_dbapi_error
    reraise(exc_type, exc_info[1], exc_info[2])
  File "/Users/skyfall/Documents/workspace/test/python/pyathena/vendor/pyathenajdbc/cursor.py", line 102, in execute
    result_set = self._statement.executeQuery(query)
pyathenajdbc.error.DatabaseError: java.sql.SQLException: Failed to run query
```

Is that correct code??

# Additional Data
- Python : 2.7.10
- java version : 1.8.0_111
- javac : 1.8.0_111

**Comment by laughingman7743** [comment]
Is it possible to tell me the executed code and the SQL statements?

**Comment by laughingman7743** [comment]
I released v1.0.5.
I updated reraise handling. It seems that error contents will be displayed in more detail in the stack trace, please update and try.

**Comment by SteveRoland** [comment]
Hi, laughingman7743.
I updated the PyAthenaJDBC v1.0.5 and I got the exception.

```
pyathenajdbc.error.DatabaseError: Unable to execute HTTP request: athena.ap-northeast-1.amazonaws.com
```

## solution
Just region problem. So, I add a region info.

my test code :
```
#-*- coding: utf-8 -*-
import os
import sys
import logging
from pprint import pprint

here = os.path.dirname(os.path.realpath(__file__))
sys.path.append(os.path.join(here, "vendor"))

from pyathenajdbc import connect

conn = connect(s3_staging_dir='s3://skyfallathena/result/log_analy/', region_name='us-east-1')
try:
    with conn.cursor() as cursor:
        cursor.execute("""
        SELECT ymd, cd_idx, sum(score) as score
        FROM skyfall.big_log_table
        where dt >= '2017-01-05'
        group by ymd, cd_idx
        """)
        # print(cursor.description)
        print(cursor.fetchall())
finally:
    conn.close()
```

Thank you laughingman7743.
